src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -31,24 +31,46 @@ public void promptNextPlayer(){
 
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
+        while (true){
+            while (board.getWinner() == null && !(board.isTie())){
+                board.printBoard();
+                promptNextPlayer();
+                String line = keyboardScanner.nextLine();
+                String input[] = line.split(",");
+                try {
+                    board.playMove(Integer.parseInt(input[0]), Integer.parseInt(input[1]));
+                } catch (InvalidMoveException e) {
+                    System.out.println("Invalid coordinates. Try again");
+                    promptNextPlayer();
+                }
+            }
 
-        while (board.getWinner() == null){
             board.printBoard();
-            promptNextPlayer();
+            board.scoreBoard();
+
+            if (board.isTie()) {
+                System.out.println("Tie game!");
+            }
+            else {
+                System.out.println("Player " + board.getWinner() + " has won the game!");
+            }
+
+            // Ask user if they would like to play again
+            System.out.println("Would you like to play again [y/n]?");
             String line = keyboardScanner.nextLine();
-            String input[] = line.split(",");
-            try {
-                board.playMove(Integer.parseInt(input[0]), Integer.parseInt(input[1]));
-            } catch (InvalidMoveException e) {
-                System.out.println("Invalid coordinates. Try again");
-                promptNextPlayer();
+
+            if(line.toLowerCase().equals("y")){
```

Comment:
You could use line.equalsIgnoreCase("y")

Reply:
I changed the method, I didn't know about that one!